po/nl.po

```diff
@@ -1,198 +1,121 @@
-#: data/com.github.arshubham.cipher.appdata.xml.in:80
+#: src/Views/CipherView.vala:84
 msgid "ROT13 Cipher"
 msgstr "ROT13-vercijfering"
 
-#: src/views/CipherView.vala:85
-#: data/com.github.arshubham.cipher.appdata.xml.in:82
+#: src/Views/CipherView.vala:85
 msgid "Base64 Encoding"
 msgstr "Base64-versleuteling"
 
-#: src/views/CipherView.vala:86
+#: src/Views/CipherView.vala:86
 msgid "Hash Functions"
 msgstr "Hashfuncties"
 
-#: src/views/CipherView.vala:87
-#: data/com.github.arshubham.cipher.appdata.xml.in:81
+#: src/Views/CipherView.vala:87
 msgid "ASCII Encoding"
 msgstr "ASCII-versleuteling"
 
-#: src/views/CipherView.vala:88
-#: data/com.github.arshubham.cipher.appdata.xml.in:84
+#: src/Views/CipherView.vala:88
 msgid "Vigenere Cipher"
 msgstr "Vigenere-vercijfering"
 
-#: src/views/CaesarCipherView.vala:61
+#: src/Views/CaesarCipherView.vala:61
 msgid "Number of letters to shift to the right: "
 msgstr "Aantal naar rechts te verschuiven letters: "
 
-#: src/views/VigenereCipherView.vala:66
+#: src/Views/VigenereCipherView.vala:66
 msgid "Key must be at least 1 character long"
 msgstr "De sleutel moet minimaal 1 teken bevatten"
 
-#: src/views/VigenereCipherView.vala:78
+#: src/Views/VigenereCipherView.vala:78
 msgid "Key: "
 msgstr "Sleutel:"
 
-#: src/views/VigenereCipherView.vala:80
+#: src/Views/VigenereCipherView.vala:80
 msgid "Preserve Case: "
 msgstr "Hoofdlettergevoeligheid behouden: "
 
-#: src/configs/Constants.vala:26 data/com.github.arshubham.cipher.desktop.in:6
-msgid "com.github.arshubham.cipher"
-msgstr "com.github.arshubham.cipher"
-
-#: src/configs/Constants.vala:28 data/com.github.arshubham.cipher.desktop.in:3
-#: data/com.github.arshubham.cipher.desktop.in:4
-#: data/com.github.arshubham.cipher.appdata.xml.in:7
+#: src/Configs/Constants.vala:25
 msgid "Cipher"
 msgstr "Cipher"
 
-#: src/configs/Constants.vala:31
-msgid "An simple app for encoding and decoding text"
-msgstr "Een eenvoudige toepassing voor het ver- en ontsleutelen van tekst"
-
-#: src/widgets/HeaderBar.vala:88
+#: src/Widgets/HeaderBar.vala:88
 msgid "Light background"
 msgstr "Licht thema"
 
-#: src/widgets/HeaderBar.vala:91
+#: src/Widgets/HeaderBar.vala:91
 msgid "Dark background"
 msgstr "Donker thema"
 
-#: src/widgets/HeaderBar.vala:131
+#: src/Widgets/HeaderBar.vala:131
 #, c-format
 msgid "Click to learn more about %s on Wikipedia"
 msgstr "Klik voor Wikipedia-informatie over '%s'"
 
-#: data/com.github.arshubham.cipher.desktop.in:5
-#: data/com.github.arshubham.cipher.appdata.xml.in:8
-msgid "Encode and decode text"
-msgstr "Tekst ver- en ontsleutelen"
-
-#: data/com.github.arshubham.cipher.desktop.in:12
-msgid "cipher;crypt;cryptography;"
-msgstr "vercijferen;kluis;cryptografie;"
-
-#: data/com.github.arshubham.cipher.desktop.in:16
-msgid "About Cipher"
-msgstr "Over Cipher"
-
-#: data/com.github.arshubham.cipher.appdata.xml.in:9
-msgid "Shubham Arora"
-msgstr "Shubham Arora"
+#~ msgid "com.github.arshubham.cipher"
```

Comment:
Why is this commented? Also noticed it in several other translations.

Reply:
I forgot to remove it after translation update. I'll fix this now.